azext_iot/central/services/device.py

```diff
@@ -105,23 +133,32 @@ def get_device_registration_summary(
 
     registration_summary = {status.value: 0 for status in DeviceStatus}
 
-    url = "https://{}.{}/{}".format(app_id, central_dns_suffix, BASE_PATH)
+    url = "https://{}.{}/{}?api-version={}".format(
```

Comment:
`api-version` is inserted directly in the url here versus defining it in query_parameters in other cases (and passing it in to the params arg).

Reply:
wont fix: route specified is restricted to V1.  - simpler solution